apps/docs/azure-functions/src/lib/auth.ts

```
@@ -2,89 +2,246 @@
  * Authentication Helpers
  *
  * Validates Azure AD B2C tokens for authenticated functions.
+ * Uses jose library for proper JWT validation.
  */
 
 import { HttpRequest } from "@azure/functions";
+import * as jose from "jose";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)